alembic/versions/2024_05_13_078884a6b2d2_fix_new_tech_tables.py

```diff
@@ -0,0 +1,29 @@
+"""fix_new_tech_tables
+
+Revision ID: 078884a6b2d2
+Revises: 9b188aee9577
+Create Date: 2024-05-13 11:36:49.628610
+
+"""
+
+import sqlalchemy as sa
+
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "078884a6b2d2"
+down_revision = "9b188aee9577"
```

Comment:
Make sure this one is correct before merging!

Reply:
last revision in #3202 had `9b188aee9577`

Reply:
correction, last revision is `4766684153e3`